include/z64.h

```diff
@@ -287,10 +287,41 @@ typedef struct {
     /* 0x10 */ Vec3f relativePos;
 } SoundSource; // size = 0x1C
 
+typedef enum {
+    /* 0x00 */ SKYBOX_NONE,
+    /* 0x01 */ SKYBOX_NORMAL_SKY,
+    /* 0x02 */ SKYBOX_BAZAAR,
+    /* 0x03 */ SKYBOX_OVERCAST_SUNSET,
+    /* 0x04 */ SKYBOX_MARKET_ADULT,
+    /* 0x05 */ SKYBOX_CUTSCENE_MAP,
+    /* 0x06 */ SKYBOX_HOUSE_LINK,
+    /* 0x09 */ SKYBOX_MARKET_CHILD_DAY = 9,
+    /* 0x0A */ SKYBOX_MARKET_CHILD_NIGHT,
+    /* 0x0B */ SKYBOX_HAPPY_MASK_SHOP,
```

Comment:
for this in particular decimal does make more sense, but it's a case by case basis imo

Reply:
Sorry I forgot to address this comment.
In general I believe we prefer hex, since as Roman has said before the comments are mostly for matching up with existing documentation from binary hacking resources, which has all this stuff in hex.
If you think we should discuss consistency of the comments overall then by all means, any discussions are valuable.